Makefile

```diff
@@ -38,21 +38,21 @@ $(PREFIX)/fossa: $(BIN)/fossa
 
 # Building various Docker images.
 docker-base: ./docker/base/Dockerfile
-	sudo docker build -t quay.io/fossa/fossa-cli-base -f ./docker/base/Dockerfile .
+	docker build -t quay.io/fossa/fossa-cli-base -f ./docker/base/Dockerfile .
```

Comment:
These are `sudo` for Linux development. You might want to add MacOS-specific targets without `sudo`.

Reply:
I do use Linux - sudo is only necessary if your user isn't part of the docker group, this is true of Linux and MacOS

    sudo groupadd docker
    sudo gpasswd -a $USER docker

c.f. https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo/477554#477554

Reply:
That's _true_, but I'm not sure that "being in the `docker` group" a good assumption.

1) Being in the `docker` group is effectively equivalent to being root, and I want to discourage this behaviour as much as possible.
2) You're effectively root anyway, so `sudo docker ...` is only more typing, not actually less secure.